broker-core/src/main/java/io/zeebe/broker/exporter/debug/DebugLogExporter.java

```diff
@@ -83,6 +83,8 @@ public void log(String message, Object... args) {
       case ERROR:
         log.error(message, args);
         break;
+      default:
+        throw new IllegalStateException("Unexpected value: " + logLevel);
```

Comment:
I guess that throwing an exception here would spam our log :sweat_smile: We could check the log level at `configure(..)`. In case it is unknown, then print a warning log statement and use the default level (e.g. DEBUG).